insights/v1alpha1/techpreview.datagather.testsuite.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "[Stable] DataGather"
+crd: 0000_10_01_datagather.crd.yaml
+tests:
+  onCreate:
+  - name: Should be able to create a minimal DataGather
+    initial: |
+      apiVersion: insights.openshift.io/v1alpha1
+      kind: DataGather
+      spec: {} # No spec is required for a DataGather
```

Comment:
Is this desired? As this is a new CR, we can introduce a minimal spec if there's something you'd expect to be here.
What happens when no spec if provided, is that ok?

Reply:
I think it is. I think it makes sense. No spec means that the default values are used, which means that no data policy is applied and all available gatherers are run.